project/utils/mri_plot.py

```diff
@@ -56,7 +56,7 @@ def plot_all_slices_from_array(predicted, case_index=0):
             if unique_values.numel() > 1:  
                 show = True
                 print(f'Case {case_index} | Slice {j}: ', unique_values)
-            ax.imshow(prediction[1][j].squeeze(0).cpu(), cmap='gray')
+            ax.imshow(prediction[0][j].squeeze(0).cpu(), cmap='gray')
```

Comment:
Why change the index?

Reply:
Di ko na kasi ginagamit yung mri_plot. Originally sa ViT yung index 1 is yung prediction and 0 is yung ground truth.   pero ngayon sa final version sariling print function na ginagamit ko. yug skulll stripping notebook ni kess yan lang alam ko yung gumagamit ng mri_plot so rinevert ko siya based sa original niyang gawa.

Reply:
Okay so retain ko to.

Reply:
👌